proto/teleport/web/README.md

```diff
@@ -0,0 +1 @@
+Protos used only by JavaScript servers and clients.
```

Comment:
I created `proto/teleport/web` because putting `pty_host_service.proto` into `proto/teleport/lib/teleterm` didn't sit right with me – `lib/teleterm` has no idea that such service even exist and will not use it.